src/apurvaJun20/GradingSystem.java

```diff
@@ -0,0 +1,57 @@
+package apurvaJun20;
+
+public class GradingSystem
+{
+
+	double sum(double num1,double num2,double num3) {
+		return num1+num2+num3;
+	}
+
+	double averageSum(double num1,double num2,double num3) {
+		return (sum(num1,num2,num3)/3);
+	}
+
+	static boolean isAnswerAboveExpecation(double average) {
+	if(average>50)
+		return true;
+     else 
+    	 return false;
+	}
+
+	static boolean isEligible(double average)
+	{
+	if(average>50 || average%2==0)
```

Comment:
Please take care of indentation from the next assignment.

Reply:
Sure, Thank you!!